src/main/java/org/phoebus/olog/entity/Log.java

```diff
@@ -384,8 +384,16 @@ public LogBuilder() {
             } else {
                 this.source = new StringBuilder();
             }
-            this.description = new StringBuilder(log.getDescription());
-            this.title = new StringBuilder(log.getTitle());
+            if (log.getDescription() != null) {
+                this.description = new StringBuilder(log.getDescription());
+            } else {
+                this.description = new StringBuilder("No Description");
+            }
+            if (log.getTitle() != null) {
+                this.title = new StringBuilder(log.getTitle());
+            } else {
+                this.title = new StringBuilder("No Title");
```

Comment:
Not sure how good this is

With the above changes now we cannot check for when description of title are empty.

I think it is better to have a empty string 
`this.description = new StringBuilder("");`